dev/io.openliberty.checkpoint_fat_crac/fat/src/io/openliberty/checkpoint/fat/crac/CRaCResourceFailCheckpointTest.java

```diff
@@ -76,6 +76,12 @@ public void setUp() throws Exception {
                                           server.waitForStringInLogUsingMark("TESTING - beforeCheckpoint " + 2, 0));
                             assertNotNull("beforeCheckpoint not called",
```

Comment:
Just a small nit, line 69, ```' message not found in log before rerstore",```

restore instead of rerstore

Reply:
I pushed up another commit that fixes this and all the other places that got it copy-pasted to in this test bucket.